daemon/graphdriver/quota/types.go

```diff
@@ -1,6 +1,7 @@
 // +build linux
 
 package quota // import "github.com/docker/docker/daemon/graphdriver/quota"
+import "sync"
```

Comment:
I'm surprised this passed the gofmt check.
Can we move add a whitespace line above import?

Reply:
Sure.

I am surprised as well -- it was auto-inserted by `goimports` which is a superset of `gofmt` -- maybe it's a corner case which they don't handle, I'll take a look.

Reply:
https://github.com/golang/go/issues/33413